sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventGrid.java

```diff
@@ -189,14 +189,9 @@ private String buildSourceMediaText(EventIndexDto event) {
 		return (srcMediaWebsite != null ? srcMediaWebsite : "") + " " + (srcMediaName != null ? "(" + srcMediaName + ")" : "");
 	}
 
-	public void setContactCountMethod(boolean SourceInEvent) {
-		if (SourceInEvent) {
-			getColumn(EventIndexDto.CONTACT_COUNT_SOURCE_IN_EVENT).setHidden(false);
-			getColumn(EventIndexDto.CONTACT_COUNT).setHidden(true);
-		} else {
-			getColumn(EventIndexDto.CONTACT_COUNT).setHidden(false);
-			getColumn(EventIndexDto.CONTACT_COUNT_SOURCE_IN_EVENT).setHidden(true);
-		}
+	public void setContactCountMethod(EventContactCountMethod method) {
+		getColumn(EventIndexDto.CONTACT_COUNT_SOURCE_IN_EVENT).setHidden(!(method == EventContactCountMethod.COUNT_ALL_CONTACTS));
```

Comment:
Don't these two code lines mean that both columns are displayed when the count method is "all" and none of them is displayed when the count method is "source case in event"?